pyproject.toml

```
@@ -2,7 +2,7 @@
 requires = [
     "setuptools",
     "cython>=0.25,<3.0",
-    "spacy>=3.2<4.0",
+    "spacy>=3.2,<3.4",
```

I'm not sure if this is necessary at build time since it's only for cython sources, do you think we could keep the previous constraint?
```suggestion
    "spacy>=3.2,<4.0",
```

---

Indeed that is not required, I reverted it :)